sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -845,7 +834,7 @@ public TextAnalyticsServiceVersion getServiceVersion() {
 
     /**
      * Returns a sentiment prediction, as well as sentiment scores for each sentiment class (Positive, Negative, and
-     * Neutral) for the document and each sentence within i
+     * Neutral) for the document and each sentence within it.
```

Comment:
Let's make sure that the docstring here specifies that these are confidence scores, not sentiment scores.